tests/structuredheadings/formatcombo-boundaries.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable */
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: wysiwygarea,structuredheadings,toolbar,basicstyles,dialog,richcombo,undo */
+/* eslint-enable */
+
+// Clean up all instances been created on the page.
+(function () {
+  var comboName = "NumFormats";
+
+  bender.editor = {
+    allowedForTests: "h1; h2; p"
+  };
+
+  bender.test({
+    setUp: function () {
+      //Anything to be run before each test if needed
+    },
+
+    tearDown: function () {
+      var editor = this.editorBot.editor;
+      editor.plugins.structuredheadings.currentScheme = "1.1.1.1.1.";
+    },
+
+    "two paragraphs, one triple clicked and made a heading": function () {
+      var bot = this.editorBot;
+      bot.editor.plugins.structuredheadings.currentScheme = "1. a. i. a. i.";
+      // this is the odd selection that occurs if you
+      // triple click "foo"
+      bot.setHtmlWithSelection("<p>[foo</p><p>]bar</p>");
```

Comment:
I always forget how slick this string parsing method of setting selections is. I wish I had thought of it for WYMeditor testing. Would have saved me a ton of annoying index-counting.